internal/domain/application_oidc.go

```diff
@@ -11,12 +11,16 @@ import (
 
 const (
 	http                          = "http://"
+	httpLocalhost                 = "http://localhost"
```

Comment:
Now that you've introduced the localhost versions (`localhost`, `127.0.0.1`, `[::1]` and `[0:0:0:0:0:0:0:1]`) without `/`, I don't believe we need all the other variations.

I believe this is enough?

```go
const (
	http               = "http://"
	httpLocalhost      = "http://localhost"
	httpLoopbackV4     = "http://127.0.0.1"
	httpLoopbackV6     = "http://[::1]"
	httpLoopbackV6Long = "http://[0:0:0:0:0:0:0:1]"
	https              = "https://"
)
```